packages/use-subscription/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "private": true,
+  "name": "use-subscription",
+  "description": "Reusable hooks",
+  "version": "16.8.6",
```

Comment:
Something I didn't put a lot of thought into, but may be worth discussing briefly: Do we want to version-lock this package with React? Or start it off as v1?

Reply:
Hmm, maybe it's time to start publishing stuff like this under an `@react` scope so it's more obvious that it's owned by the React team?